pkgs/development/python-modules/nanobind/default.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchFromGitHub,
+  cmake,
+  eigen,
+  ninja,
+  scikit-build,
+  pytestCheckHook,
+  numpy,
+  scipy,
+  torch,
+  jax,
+  jaxlib,
+  tensorflow,
+  setuptools,
+}:
+buildPythonPackage rec {
+  pname = "nanobind";
+  version = "1.9.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "wjakob";
+    repo = pname;
```

Comment:
use of pname is considered an anti-pattern https://github.com/NixOS/nixpkgs/issues/277994

Reply:
From skimming through the issue, it is not obvious to me what the problem is and how the fix would look like for nanobind. Do you have any insights on this?

Reply:
basically it's about usage of pname ,when it's not really linked.

eg in `repo = pname,` if someone forks a packages and keeps name